packages/inference/src/providers/hyperbolic.ts

```diff
@@ -16,3 +14,31 @@ export const HYPERBOLIC_API_BASE_URL = "https://api.hyperbolic.xyz";
  *
  * Thanks!
  */
+import type { ProviderConfig, UrlParams, HeaderParams, BodyParams } from "../types";
+
+const HYPERBOLIC_API_BASE_URL = "https://api.hyperbolic.xyz";
+
+const makeBody = ({ args, model, taskHint }: BodyParams): Record<string, unknown> => {
+	return {
+		...args,
+		...(taskHint === "text-to-image" ? { model_name: model } : { model }),
+	};
+};
+
+const makeHeaders = ({ accessToken }: HeaderParams): Record<string, string> => {
+	return { Authorization: `Bearer ${accessToken}` };
+};
+
+const makeUrl = ({ baseUrl, taskHint }: UrlParams): string => {
+	if (taskHint === "text-to-image") {
+		return `${baseUrl}/v1/images/generations`;
+	}
+	return `${baseUrl}/v1/chat/completions`;
+};
+
+export const hyperbolicConfig: ProviderConfig = {
```

Comment:
Let's use screaming snake case for exported constants
```suggestion
export const HYPERBOLIC_CONFIG: ProviderConfig = {
```

Reply:
Wasn't sure if it was considered as a constant of not (given it has methods). Will do!